src/time_zone_lookup.cc

```diff
@@ -152,6 +162,9 @@ time_zone local_time_zone() {
   free(localtime_env);
   free(tz_env);
 #endif
+#if defined(__APPLE__)
```

Comment:
Given that the conditional above is `#elif`, might as well make this one too.

Reply:
Sure. Done.